frontend/LockedIn/components/AddChallenge.js

```diff
@@ -35,77 +42,78 @@ const AddChallenge = ({ navigation }) => {
 
   const createAndUploadChallenge = async () => {
     if (!selectedImage) {
-      alert('Please select an image for the challenge.');
+      alert("Please select an image for the challenge.");
```

Comment:
should change alerts too a modal in the future

Reply:
Ill open a ticket for this